lib/rules/best-practises/no-empty-blocks.js

```diff
@@ -16,32 +20,53 @@ const meta = {
           description: 'empty block in constructor without parent initialization',
           code: 'constructor () {}',
         },
+        {
+          description: 'empty modifier with only _ (when allowEmptyModifiers is false)',
+          code: 'modifier onlyOwner { _; }',
+        },
       ],
       good: [
         { description: 'empty receive function', code: 'receive () external {}' },
         { description: 'empty fallback function', code: 'fallback () external {}' },
         {
-          description: 'empty constructor with member initialization list',
-          code: 'constructor(uint param) Foo(param) Bar(param*2) {}',
+          description: 'empty constructor with parent init',
```

Comment:
why did you choose this other wording? if we want to be consistent with the solidity docs... there's not a clear name for this, as there is for C++ 
They seem to refer to it as  ...[modifier-style base constructor invocation](https://docs.soliditylang.org/en/latest/contracts.html#arguments-for-base-constructors) ?